app/graphql/types/user_type.rb

```diff
@@ -14,6 +14,14 @@ def authorized?(_object, _args, context)
         context[:current_school_admin].present?
       end
     end
+    field :current_standing_name, String, null: true
```

Comment:
Let's not create custom fields, let assosiate the model `standing`.
Let the user decide what they want to query.

Reply:
It will make the code complex because currently, in the UI the data required is only the current standing name in `CoursesStudents__StudentOverlay` 
**- The current standing can be either the latest user standing. If user do not have any user standing records, then it will be the school's default standing.**